AdyenActions/Actions/RedireactableAwaitAction.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Describes an action in which the SDK is waiting for user action.
```

Comment:
Maybe we can mention the redirect part of this new action.

Reply:
Good one!